### ISSUE TITLE
AudioContext is suspended on iOS until another user interaction happens

### ISSUE THREAD

**Issue by gugiserman** [goldfire/howler.js#1031]
It's a React + Redux app with a scenario as follows:

1. User clicks on 'Play'
2. Click handler function dispatches async redux action that instantiates a Howl with a single .mp3 file as source, and sets `html5: true` to it.
3. MP3 file is loaded but the sound doesn't start playing at all, not with `autoplay`, not by manually calling `.play` on that specific sound, etc
4. Click on "pause" then "play" again and playback starts (same Howl instance, same source, same AudioContext I guess).

I've tried several attempts to resume `Howler.ctx` from multiple triggers, including `touchend` events. Also the global `mobileAutoEnable` is set to true. I also tried playing an empty buffer before my sound, and nothing...

If I simply instantiate the Howl inside the click handler function, it works. The problem only happens when trying to instantiate it from somewhere else, i.e.: an action (redux action).

It also starts playback if I set it to `mute: true`. But as soon as I try to unmute any sound, playback stops and waits for another user interaction to resume.

---

Update: I forgot to mention this issue is on iOS only. Works on desktop and Android, but not on iPhones (haven't tested it on iPad yet).

---

Another update: This is how I tried to silently play something, and it actually works. This is fired from the very first `touchend` event on the page, and `Howler.ctx.state === 'running'` afterwards, but still no playback when the actual requested sound is loaded when the user clicks "Play".
```js
const buffer = Howler.ctx.createBuffer(1, 1, 22050)
const source = Howler.ctx.createBufferSource()
source.buffer = buffer
source.connect(Howler.ctx.destination)
source.noteOn(0)
```

Any ideas?

**Comment by gugiserman** [comment]
@scottanderson42 Unfortunately I couldn't get your PR to work, so I left a comment there #1008

**Comment by goldfire** [comment]
This should be fixed in the upcoming v2.1.0, but let me know if that doesn't solve your problem and I'll reopen.

**Comment by goldfire** [comment]
@gugiserman The audio pool won't get filled until there is a tap/click after the `Howl` is setup. It can't be setup in anything other than a user interaction.
